webpack.config.js

```diff
@@ -4,13 +4,14 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
   mode: NODE_ENV,
 
   entry: {
-    site: './assets/js/main.js',
+    main: './assets/js/main.js',
+    contact: './assets/js/contact.js',
   },
 
   target: ['web', 'es5'],
 
   output: {
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
is this the right name? or is this some sort of placeholder?

Reply:
> is this the right name? or is this some sort of placeholder?

It's substituted by Webpack using the key from the `entry` object above.

See: https://webpack.js.org/configuration/output/#template-strings

Related (exact example): https://webpack.js.org/concepts/output/#multiple-entry-points